mangaki/mangaki/views.py

```diff
@@ -383,56 +385,107 @@ def get_context_data(self, **kwargs):
         return context
 
 
-def get_profile(request, username):
-    user = get_object_or_404(User, username=username)
-    is_shared = user.profile.is_shared
+def remove_all_anon_ratings(request):
```

Comment:
This should only have effect with `POST` requests (+ caller HTML should change to handle this). Let's try to avoid non-idempotent `GET` requests when we can.